resources/web/panel/css/style.css

```diff
@@ -808,3 +808,26 @@ input, textarea {
     border-color: grey;
 }
 
+
```

Comment:
i wasn't sure where in the css it would go so i just stuck it there for you guys to make sure it goes in a neat spot :)